desktop/core/src/desktop/views.py

```diff
@@ -390,14 +390,14 @@ def dump_config(request):
   return render("dump_config.mako", request, {})
 
 
-@hue_admin_required
+@login_notrequired
```

Comment:
Does this `@login_notrequired` make the threaddump available to everyone regardless of auth? Or is there another config knob that keeps it disable most of the time?
I am worried that this could be a big security issue if the threaddump is available to anyone.

Reply:
@wing2fly , I dont think there is one on the newer releases.  We used to have one but that was removed in favor of UI.

@amitsrivastava ,  I just used that command I had it saved for another issue and used the same here.